jobs/base_model_validator.py

```diff
@@ -115,6 +115,25 @@ def process(self, input_model):
             yield errors.ModelMutatedDuringJobError(model)
 
 
+class ValidateCommitType(beam.DoFn):
```

Comment:
Could you also please introduce the `BaseSnapshotMetadataModelValidator` that will use this validator?

Reply:
@vojtechjelinek `BaseSnapshotMetadataModelValidator` or `BaseCommitLogEntryModelValidator`?

Reply:
I guess `BaseCommitLogEntryModelValidator`, but then this class can also be used in `BaseSnapshotMetadataModelValidator`.

Reply:
@vojtechjelinek Aren't both models `BaseCommitLogEntryModel` and `BaseSnapshotMetadataModel` different? I have used MockCommitLogEntryModel in my tests. Am I wrong with this approach?

Reply:
Yeah, but both have `commit_type` :slightly_smiling_face:

Reply:
Oh okay. I'll try to write `BaseSnapshotMetadataModelValidator`. Thanks!

Reply:
I think using the most-base class is the best too.

Reply:
@vojtechjelinek @brianrodri I am confused and think I might have understood this. I have updated the code, please take a look and verify whether it is right or wrong. Thanks!